feature/src/main/java/com/gdg/feature/calendar/CalendarRoute.kt

```diff
@@ -1,51 +1,394 @@
+    selectedDate: LocalDate,
+    onDateSelected: (LocalDate) -> Unit,
     paddingValues: PaddingValues = PaddingValues()
 ) {
+    var currentMonth by remember { mutableStateOf(YearMonth.now()) }
+
     Column(
         modifier = Modifier
+            .fillMaxSize()
             .padding(paddingValues)
             .navigationBarsPadding()
-            .statusBarsPadding()
-            .fillMaxSize(),
-        verticalArrangement = Arrangement.Center,
+            .statusBarsPadding(),
+        verticalArrangement = Arrangement.Top,
         horizontalAlignment = Alignment.CenterHorizontally
     ) {
-        Text(
-            text = "Calendar Screen"
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 16.dp, vertical = 12.dp),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Column(modifier = Modifier.weight(1f)) {
+                Text(
+                    text = buildAnnotatedString {
+                        withStyle(style = SpanStyle(color = CrowdZeroTheme.colors.gray900)) {
+                            append(stringResource(R.string.calender_header_1))
+                        }
+                        withStyle(style = SpanStyle(color = CrowdZeroTheme.colors.green700)) {
+                            append(stringResource(R.string.calender_header_2))
+                        }
+                        withStyle(style = SpanStyle(color = CrowdZeroTheme.colors.gray900)) {
+                            append(stringResource(R.string.calender_header_3))
+                        }
+                    },
+                    style = CrowdZeroTheme.typography.h2Bold
+                )
+
+                Spacer(modifier = Modifier.height(4.dp))
+
+                Text(
+                    text = stringResource(R.string.calender_sub_header_1),
+                    style = CrowdZeroTheme.typography.h3Regular,
+                    color = CrowdZeroTheme.colors.gray700
+                )
+                Text(
+                    text = stringResource(R.string.calender_sub_header_2),
+                    style = CrowdZeroTheme.typography.h3Regular,
+                    color = CrowdZeroTheme.colors.gray700
+                )
+            }
+
+            Image(
+                painter = painterResource(R.drawable.ic_alert),
+                contentDescription = null,
+                modifier = Modifier.size(105.dp)
+            )
+        }
+
+        // 캘린더 컴포넌트
+        CalendarComponent(
+            currentMonth = currentMonth,
+            selectedDate = selectedDate,
+            onMonthChange = { currentMonth = it },
+            onDateSelected = onDateSelected
         )
+
+        Spacer(
+            modifier = Modifier
+                .height(1.dp)
+                .fillMaxWidth()
+                .background(CrowdZeroTheme.colors.gray500)
+        )
+
+        Spacer(modifier = Modifier.height(25.dp))
```

Comment:
P1: 구분선은 HorizontalDivider 사용해주세요
```suggestion
        HorizontalDivider(
            modifier = Modifier
                .fillMaxWidth()
                .padding(top = 20.dp, bottom = 23.dp),
            thickness = 1.dp,
            color = CrowdZeroTheme.colors.gray500
        )
```